docker/integtest/6-receive-service-email.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+scriptdir="$(dirname "$0")"
+in_dir="${scriptdir}/files"
+out_dir="${scriptdir}/files/test.out"
+mkdir -p "${out_dir}"
+# shellcheck disable=SC1090
+. "${scriptdir}/utils.sh"
+
+email_to_receive="${in_dir}/inbound-email.mime"
```

Comment:
To make this a fully representative integration test we should change the email content to be an actual bot email (i.e. sent to `echo@bot.lokole.ca`). You should be able to make a copy of the MIME file and modify the metadata accordingly.

Reply:
will do!